app/actions/actions.js

```diff
@@ -1,12 +1,14 @@
 import { ADD_PRODUCER_DATA, ADD_CONSUMER_DATA } from '../constants/actionTypes';
 
+// Action creator that describes and generates the actions to be dispatched for Producer
```

Comment:
remove the words "describes and". this function isn't really describing anything